documentation/library_guide/macros.rst

```diff
@@ -64,11 +64,34 @@ Macro                              Description
 ---------------------------------- ------------------------------
 ``ONEDPL_USE_TBB_BACKEND``         This macro controls the use of |onetbb_long| or
                                    |tbb_long| for parallel policies.
-                                   When the macro is set to 0, algorithms with the ``par`` and ``par_unseq`` policies are only
-                                   executed by the calling thread. This is recommended for code that should not depend on the
-                                   presence of the |onetbb_short| or |tbb_short| library. When the macro is not defined (by default)
-                                   or evaluates to a non-zero value,
+                                   When the macro is set to 0, algorithms with the ``par`` and ``par_unseq`` policies are do not
+                                   consider this backend to use and try to find the "next" backend according to the priotiry.
```

Comment:
Why do you use quotes around `"next"`?

Reply:
Because we never defined what _next_ means. It emphasizes that it's "some kind of next"